docs/mindsdb-docs/docs/sql/create/predictor.md

```diff
@@ -114,6 +114,23 @@ model.args='{"key": value}'
 ;
 ```
 
+Module options:
+
+| Module          | Description     |
+| --------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [**BaseMixer**](https://lightwood.io/mixer.html#mixer.BaseMixer)     | Base class for all mixers.                                                                                                                                        |
+| [**LightGBM**](https://lightwood.io/mixer.html#mixer.LightGBM)    |                                                                                                                                                                   |
+| [**LightGBMArray**](https://lightwood.io/mixer.html#mixer.LightGBMArray) | LightGBM-based model, intended for usage in time series tasks.                                                                                                    |
+| [**NHitsMixer**](https://lightwood.io/mixer.html#mixer.NHitsMixer)    | Wrapper around a MQN-HITS deep learning model.                                                                                                                    |
+| [**Neural**](https://lightwood.io/mixer.html#mixer.Neural)        | The Neural mixer trains a fully connected dense network from concatenated encoded outputs of each of the features in the dataset to predicted the encoded output. |
+| [**ProphetMixer**](https://lightwood.io/mixer.html#mixer.ProphetMixer)  | This mixer is a wrapper around the popular time series library sktime.                                                                                            |
+| [**Regression**](https://lightwood.io/mixer.html#mixer.Regression)    | The Regression mixer inherits from [scikit-learn’s Ridge class](https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.Ridge.html)                |
+| [**SkTime**](https://lightwood.io/mixer.html#mixer.SkTime)        | This mixer is a wrapper around the popular time series library sktime.                                                                                            |
+| [**Unit**](https://lightwood.io/mixer.html#mixer.Unit)         |                                                                                                                                                                   |
```

Comment:
Special mixer that passes along whatever prediction is made by the target encoder without modifications. This is useful for single-column predictive scenarios that may involve complex and/or expensive encoders (e.g. free-form text classification with transformers).